scripts/reindex.py

```diff
@@ -96,9 +96,10 @@ def main(argv: List[str]):
     if args.purge:
         queue_manager = Queues()
         queues = dict(queue_manager.azul_queues())
+        queues = {key: val for key, val in queues.items() if key in config.work_queue_names}
```

Comment:
```suggestion
        # FIXME: eliminate need for filtering (https://github.com/DataBiosphere/azul/issues/1448)
        queues = {k: v for k, v in queues.items() if k in config.work_queue_names}
```

AFAICT, there's precedence for using `k` and `v` all over the codebase. Please stick with existing conventions as outlined in the contributions guide. Let me know if I am wrong about that.

Reply:
No you are correct. I had thought you previously recommended the opposite but was mistaking a different situation that doesn't apply here inside a dict comprehension.